src/BiRRT.hpp

```diff
@@ -36,6 +36,14 @@ namespace RRT
             return _goalTree;
         }
 
+        bool isDynamic() const {
```

Comment:
These should probably be named similarly since they reference the same property.  Maybe isAscEnabled and setAscEnabled?